### ISSUE TITLE
Every Time a new TableViewController is created, it calls the server twice to refresh its viewModel

### ISSUE THREAD

**Issue by alexbbrown** [metabolist/metatext#266]
**Describe the bug**

This is a network performance bug.  Each time a new TableViewController is created, it refreshes its model twice.

This happens in at least two circumstances, for a variety of content types in the app. 

1. The first time each of the 3 timelines are tapped - the server is called twice
2. Each time a view controller with a table in it is pushed
   A. When announcements button is pressed 
   B. When a thread is opened
   C. When a profile is loaded

This is probably a bad idea, it consumes server time and cellular network up to 2x as fast as is necessary

**To Reproduce**
1. set a breakpoint on each call to `viewModel.request` in `TableViewController.swift`
2. tap on a thread

**Expected behavior**

the server refresh should be called once. 
the server refresh IS called twice

I looked at what happens when you tap notifications, and saw that it calls the announcement API *twice* each time, once in viewDidLoad and once in viewWillAppear

```swift
    // TableViewController.swift

    override func viewDidLoad() {
        …
        viewModel.request(maxId: nil, minId: nil, search: nil) 
    }

    override func viewWillAppear(_ animated: Bool) {
        super.viewWillAppear(animated)

        refreshIfAble()
    }
…
    func refreshIfAble() {
        if viewModel.canRefresh {
            viewModel.request(maxId: nil, minId: nil, search: nil)
        }
    }
```

**Recommended**
Avoid refreshing twice.  Probably simplest to remove it from viewDidLoad?

**Screenshots**

**(please complete the following information):**
 - Device: iPhone 12
 - OS: iOS 16.2
 - App Version 1.7

**Additional context**

I have a debug printing patch I can share for private use.

**Comment by jzzocc** [comment]
Thank you, this is being immediately addressed.

**Comment by jzzocc** [comment]
Fixed in v1.7.1
